pb/lmsysmd/load/data/v1/data.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+
+package lmsysmd.load.data.v1;
+
+import "buf/validate/validate.proto";
+import "lmsysmd/load/model/v1/model.proto";
+
+service DataService {
+  rpc BatchCreateData(BatchCreateDataRequest) returns (BatchCreateDataResponse) {
+    option idempotency_level = IDEMPOTENT;
+  }
+}
+
+message BatchCreateDataRequest {
+  string google_sheet_url = 1 [
+    (buf.validate.field).required = true,
+    (buf.validate.field).string.uri = true,
+    (buf.validate.field).string.min_len = 2,
+    (buf.validate.field).string.max_len = 150
+  ];
+}
+
+message BatchCreateDataResponse {}
+
+// The fields are based on the columns in https://docs.google.com/spreadsheets/d/1MWVXNHXgLdTHHTVmpi9HmQ3DRV6uU3CWfHdxshUeEUU/edit?gid=0#gid=0.
+message Data {
+  uint64 experiment_id = 1 [
+    (buf.validate.field).required = true,
+    (buf.validate.field).uint64.gt = 0
+  ];
+  lmsysmd.load.model.v1.ModelId model_id = 2 [
+    (buf.validate.field).required = true,
+    (buf.validate.field).enum.defined_only = true
+  ];
+  enum CaseCategory {
+    CASE_CATEGORY_UNSPECIFIED = 0;
+    CASE_CATEGORY_BR = 1;
```

Comment:
Could you use the full name?

Reply:
I used the abbreviations because that's what the lab folks will be inputting not the full name. Could we just add a legend in the operations doc instead? Otherwise I can ask them to switch to full names.

Reply:
You can code a converter in the go code to map their abbreviations to the enum variant with full name